dj_hr/data/dj.xml

```diff
@@ -0,0 +1,22 @@
+<odoo>
+  <record model="dj.compilation" id="dj_hr_base">
+    <field name="name">HR</field>
+    <field name="genre_id" ref="dj_core.dj_custom_genre" />
+    <field name="core" eval="True" />
+  </record>
+
+  <record model="dj.song" id="dj_song_resource_calendar">
+    <field name="compilation_id" ref="dj_hr_base"/>
+    <field name="model_id" ref="resource.model_resource_calendar" />
```

Comment:
correct me if I'm wrong: aren't these auto created? Don't we have already some resources that do not match our xids and will get duplicated?
If yes, then you should add also a song to generate xids for existing records.

Reply:
@simahawk how do you mean? All the autocreated ones already have xmlids.

Reply:
usually not. Clarification: "autocreated" means that they are created by odoo via create/write override and not via xml files or csv files. A good example are default locations for warehouse or sequences.
So, in 99.99999999% of the cases they DO NOT have a XID and you are going to export them and recreate them.
Either you exclude them but you always hard to know which ones or you put a song before of type ["generate_xmlids"](https://github.com/camptocamp/odoo-dj/blob/11.0/base_dj/config.py#L103). 
This will run before the `load_csv` one and will assign proper xids to exiting records IF they do not have a XID.

Reply:
@simahawk any changes here?